app/views/admin/profile_fields/_add_group_modal.html.erb

```diff
@@ -1,4 +1,4 @@
-<button class="btn btn-secondary" type="button" onclick="showAddGroupModal()">
+<button class="btn btn-primary" type="button" onclick="showAddGroupModal()">
```

Comment:
This seems a little random, was this change intended?

Reply:
Yes 🙈, I just slipped it in sneakily because I noticed that all the other add buttons on the admin interface had a primary class name.